faf-commons-data/src/main/java/com/faforever/commons/replay/RawReplayDataReader.java

```diff
@@ -0,0 +1,84 @@
+package com.faforever.commons.replay;
+
+import com.google.common.io.BaseEncoding;
+import com.google.common.io.ByteArrayDataInput;
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonSyntaxException;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+
+/**
+ * Utility class for unpacking replay data.
+ */
+public class RawReplayDataReader {
```

Comment:
This class should not be a static class. It wil cause a lot of problems and works against our classic Spring dependency injection pattern.